services/tests/literpc_tpu_quic_server_integrationtest.rs

```diff
@@ -0,0 +1,444 @@
+use futures::future::join_all;
+use log::{debug, error, info, trace, warn};
+use quinn::TokioRuntime;
+use serde::de::Unexpected::Option;
+use solana_lite_rpc_core::quic_connection_utils::QuicConnectionParameters;
+use solana_lite_rpc_core::structures::identity_stakes::IdentityStakes;
+use solana_lite_rpc_core::tx_store::empty_tx_store;
+use solana_lite_rpc_services::tpu_utils::tpu_connection_manager::TpuConnectionManager;
+use solana_rpc_client::rpc_client::SerializableTransaction;
+use solana_sdk::hash::Hash;
+use solana_sdk::instruction::Instruction;
+use solana_sdk::message::Message;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::signature::{Keypair, Signature, Signer};
+use solana_sdk::signer::keypair;
+use solana_sdk::transaction::{Transaction, VersionedTransaction};
+use solana_streamer::nonblocking::quic::ConnectionPeerType;
+use solana_streamer::packet::PacketBatch;
+use solana_streamer::quic::StreamStats;
+use solana_streamer::streamer::StakedNodes;
+use solana_streamer::tls_certificates::new_self_signed_tls_certificate;
+use std::collections::HashMap;
+use std::net::{IpAddr, Ipv4Addr, SocketAddr, UdpSocket};
+use std::ops::Deref;
+use std::path::Path;
+use std::str::FromStr;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::{Arc, RwLock};
+use std::thread;
+use std::time::{Duration, Instant};
+use tokio::runtime::{Builder, Runtime};
+use tokio::sync::broadcast;
+use tokio::sync::broadcast::error::SendError;
+use tokio::task::JoinHandle;
+use tokio::time::{interval, sleep};
+use tokio::{join, spawn, task};
+use tracing_subscriber::{filter::LevelFilter, fmt};
+
+// note: logging will be auto-adjusted
+const SAMPLE_TX_COUNT: u32 = 20;
+
+const MAXIMUM_TRANSACTIONS_IN_QUEUE: usize = 200_000;
+const MAX_QUIC_CONNECTIONS_PER_PEER: usize = 8; // like solana repo
+
+#[test] // note: tokio runtimes get created as part of the integration test
+pub fn wireup_and_send_txs_via_channel() {
+    configure_logging();
+
+    // value from solana - see quic streamer - see quic.rs -> rt()
+    const NUM_QUIC_STREAMER_WORKER_THREADS: usize = 1;
+    let runtime_quic1 = Builder::new_multi_thread()
+        .worker_threads(1)
+        .thread_name("quic-server")
+        .enable_all()
+        .build()
+        .expect("failed to build tokio runtime for testing quic server");
+
+    // lite-rpc
+    let runtime_literpc = tokio::runtime::Builder::new_multi_thread()
+        // see lite-rpc -> main.rs
+        .worker_threads(16) // note: this value has changed with the "deadlock fix" - TODO experiment with it
+        .enable_all()
+        .build()
+        .expect("failed to build tokio runtime for lite-rpc-tpu-client");
+
+    let literpc_validator_identity = Arc::new(Keypair::new());
+    let udp_listen_socket = UdpSocket::bind("127.0.0.1:0").unwrap();
+    let listen_addr = udp_listen_socket.local_addr().unwrap();
+
+    let (inbound_packets_sender, inbound_packets_receiver) = crossbeam_channel::unbounded();
```

Comment:
Cross beam usually have many issues and does not run well with tokio. Make it mpsc::UnboundSender.

Reply:
ma :) solana-quic-streamer brings in that dependency

Reply:
Ahh yes sad.